src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -127,6 +128,36 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    private static String encryptWithoutSpecifyingAlgorithmParams(String value) {
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
+            byte[] encrypted = cipher.doFinal(value.getBytes());
+            return StringHelper.byteArrayToHexString(encrypted);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Understood; meant that you can change the method's signiture to state that it explicitly throws those exceptions, like so:
```
f() throws NullPointerException, InvalidParametersException {
}
```

That would fix that unhandled exception error.